typeql/reasoner/attribute-attachment.feature

```diff
@@ -228,3 +302,10 @@ Feature: Attribute Attachment Resolution
     Then verify answer size is: 1
     Then verify answers are sound
     Then verify answers are complete
+    Given reasoning query
```

Comment:
what's this for?

Reply:
Feels more complete to have interacting value-constraints.